src/utils/main.js

```diff
@@ -0,0 +1,53 @@
+import { constants } from '../app/config';
+
+export const log = (message, type = 'log') => {
+  if (!constants.DEBUG) {
+    return;
+  }
+  if (type === 'log') {
+    // eslint-disable-next-line no-console
+    console.log(message);
+  }
+  if (type === 'error') {
+    // eslint-disable-next-line no-console
+    console.error(message);
+  }
+};
+
+export function randomNumber (min, max) {
+  min = Math.ceil(min);
+  max = Math.floor(max);
+  min = Math.ceil(min);
+  max = Math.floor(max);
+  return Math.floor(Math.random() * (max - min + 1)) + min;
+}
+
+export const generateRandomRgbColor = (ra, ga, ba) => {
+  const r = randomNumber(ra[0], ra[1]);
+  const g = randomNumber(ga[0], ga[1]);
+  const b = randomNumber(ba[0], ba[1]);
+  return `rgb(${r}, ${g}, ${b}`;
+};
+
+export const arrayIsInArray = (arr, subArr) => {
```

Comment:
IMO, instead of nested loops, a combination of .some() & .every() can make this function more readable (& shorter).

Reply:
what a awesome recommendation
fixed with .some()